lmms_eval/models/llama_vision.py

```diff
@@ -15,6 +15,7 @@
 from lmms_eval.api.instance import Instance
 from lmms_eval.api.model import lmms
 from lmms_eval.api.registry import register_model
+from lmms_eval.models.model_utils.load_video import read_video_pyav_pil
```

Comment:
This currently breaks as `read_video_pyav_pil` doesn't exist @Luodian

Reply:
@pufanyi

Reply:
Hello, sorry about this. It seems that this change was accidentally brought over from another branch. Thank you for pointing it out! I've reverted it in #447.